src/components/home/modals/Lease.vue

```diff
@@ -259,7 +259,7 @@ export default {
             return BigNumber(num)
         },
         coldApi: function() {
-            if (this.coldAddresses[this.coldAddress].api === 1 && BigNumber(this.coldAmount).isLessThan(BigNumber(Number.MAX_SAFE_INTEGER).dividedBy(1e8))) {
+            if (this.coldAddresses[this.coldAddress].api === 1 && (BigNumber(this.coldAmount).isLessThan(BigNumber(Number.MAX_SAFE_INTEGER).dividedBy(1e8)) || BigNumber(this.coldAmount).mod(100).isEqualTo(0))) {
```

Comment:
change to `...|| BigNumber(this.coldAmount).multipliedBy(1e8).mod(100).isEqualTo(0)`